admiral/cmd/admiral/cmd/root.go

```diff
@@ -50,14 +50,20 @@ func GetRootCmd(args []string) *cobra.Command {
 			}
 
 			service := server.Service{}
+			metricsService := server.Service{}
 			opts.RemoteRegistry = remoteRegistry
-			ret_routes := routes.NewAdmiralAPIServer(&opts)
+
+			mainRoutes := routes.NewAdmiralAPIServer(&opts)
+			metricRoutes := routes.NewMetricsServer()
 
 			if err != nil {
 				log.Error("Error setting up server:", err.Error())
 			}
 
-			service.Start(ctx, 8080, ret_routes, routes.Filter, remoteRegistry)
+			go func() {
+				metricsService.Start(ctx, 6900, metricRoutes, routes.Filter, remoteRegistry)
+			}()
+			service.Start(ctx, 8080, mainRoutes, routes.Filter, remoteRegistry)
```

Comment:
This server should be started only if metrics are enabled?

Reply:
currently implementation has 2 options. prometheus and noop. that way a 200 is always returned but the noop returns no data.